core/docs/spark-profiling-tool.md

```diff
@@ -325,87 +327,6 @@ WholeStageCodeGen Mapping:
 |1       |0    |7     |WholeStageCodegen (2)|Sort               |8           |
```

Comment:
Maybe note that WholeStageCodeGen to Node Mapping is only for CPU runs here as well (at top of this section).

Reply:
ok, i'll add a note